### PR TITLE
Add License (cc4.0) to repo root

### LINKED ISSUE #16: Provide License
We already discussed it's going to be **Creative Commons CC BY 4.0**, but I think this should also be put somewhere in the book (i.e. on the site).

### PR DESCRIPTION
**Pull request by rogierslag**
Closes https://github.com/delftswa/delftswa.github.io/issues/16

@avandeursen Ready for review!

### CODE REVIEW

**Comment by RickWieman** [comment]
I actually meant a license on the website itself, not just a license in this repo

**Comment by rogierslag** [comment]
This will also be copied to the website, it will just need a link

**Comment by RickWieman** [comment]
Check. Let's not forget about that then :blush:

**Comment by rogierslag** [comment]
That will happen in the intro text which will be written by @avandeursen. I will put it in the issue as well ;)

**Comment by RickWieman** [comment]
:+1:

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
+## creative commons
+
+# Attribution 4.0 International
```
Comment (RickWieman) [inline LICENSE.md]: These headings are rather strange; first H2, then H1 and later on H3...?

**Comment by avandeursen** [comment]
I have this covered in the introduction that I'm writing -- issue #14. Almost done.

So let's wait for a #14, and then see if that adequately addresses the license.

**Comment by RickWieman** [comment]
I like the way Arie wrote about the license in #19, and I don't think the introduction requires a link to this document (Arie references the license itself directly, which I think suffices).
I do think this `LICENSE.md` should be in the repo root, as that's more clearer from a repository point of view (i.e. not all GitHub viewers will read/see Arie's introduction).
That's basically what this PR does right now, though.

**Comment by rogierslag** [comment]
I think both PR's are a great addition to the book in the end form. On the website, the portion of @avandeursen should be sufficient, whereas on Github itself the `LICENSE.md` might be clearer.

**Comment by RickWieman** [comment]
That's what I said, right? :smile:

**Comment by rogierslag** [comment]
Yes but shorter. Arie is a very busy man ;)

**Comment by avandeursen** [comment]
Thanks for good discussion and insight. Adjusted the opening header as suggested by @RickWieman.

**Comment by RickWieman** [comment]
:+1:

**Comment by rogierslag** [comment]
:+1:

**Pull request merged**